.gitignore

```
@@ -50,7 +50,8 @@ build/.cmake/
 !/mods/minetest/
 /mods/minetest/*
 !/mods/minetest/mods_here.txt
```

Btw. whouldn't this be 
```suggestion
!/mods/mods_here.txt
```
?

---

all references to mods/minetest/ are wrong, the folder used to be called that 10 years ago

---

I guess I'll update it, when I'm already here.